### DISCUSSION TITLE
Backup data

### DISCUSSION THREAD

**Discussion by KrispyDotlessI** [chrislo27/PolyrhythmMania#48] [Ideas]
Let's imagine you have moved into another computer, but want to keep your stats intact, or even worse, if you get all your data accidentally lost and cannot recover back.

You could manually store backup datas of achievements, stats, and highscores in the game by clicking a simple button **[Create Backup Data]**, and then you can restore from a backup **[Import Backup Data]** to restart where you left off.

**Comment by chrislo27** [comment]
I do like this suggestion. I'll see if I can put it in in some capacity for the next feature release.

If you do need to move your savefiles, the directory that contains everything (including your Library levels, if you didn't change the location) is:
* Windows: `%USERPROFILE%/.polyrhythmmania`
* Linux: `~/.polyrhythmmania`

In Portable Mode, the `.polyrhythmmania` directory is alongside the launch executables.

You can then copy this folder to another device and it will load preferences, etc from there. You should do this while the game is not running.
